common/src/test/java/io/netty/util/internal/NativeLibraryLoaderTest.java

```diff
@@ -48,34 +46,28 @@ public void testFileNotFoundWithNullClassLoader() {
             fail();
         } catch (UnsatisfiedLinkError error) {
             assertTrue(error.getCause() instanceof FileNotFoundException);
-            if (getSupressedMethod != null) {
+            if (PlatformDependent.javaVersion() >= 7) {
                 verifySuppressedException(error, ClassNotFoundException.class);
             }
         }
     }
 
+    @SuppressJava6Requirement(reason = "uses Java 7+ Throwable#getSuppressed but is guarded by version checks")
     private static void verifySuppressedException(UnsatisfiedLinkError error,
             Class<?> expectedSuppressedExceptionClass) {
+        if (PlatformDependent.javaVersion() < 7) {
```

Comment:
I think we can remove this... If you really want to keep it make it an `assert`.